access-control.yaml

```diff
@@ -949,6 +949,13 @@ teams:
   - name: identus-admins
     maintainers:
       - yshyn-iohk
+    members:
+      - FabioPinheiro
+      - amagyar-iohk
+      - milosbackonja
+      - patlo-iog
+      - Dale-iohk
+      - elribonazo
   - name: identus-maintainers
     maintainers:
       - essbante-io
```

Comment:
Is one maintainer enough? What is happening is the maintainer is not available?

Reply:
It exists as a fallback. CLOWarden is the maintainer overall, followed by LF staff.